daemon/start.go

```diff
@@ -224,6 +224,15 @@ func (daemon *Daemon) containerStart(ctx context.Context, daemonCfg *configStore
 		}
 	}()
 
+	// Finish sandbox initialisation (if there's a sandbox, so not using container networking).
+	if sb, err := daemon.netController.GetSandbox(container.ID); err == nil {
+		if err := sb.CtrTaskCreated(); err != nil {
+			log.G(ctx).WithError(err).WithField("container", container.ID).
+				Error("failed to complete network configuration")
```

Comment:
Small nit; not critical (and there's tons of locations where we have this), but there's some overhead when using multiple consecutive `WitField` / `WithError` / `WithFields`. As this is new code, perhaps use `WithFields` for this case;

```suggestion
			log.G(ctx).WithFields(log.Fields{
				"container": container.ID,
				"error":     err,
			}).Error("failed to complete network configuration")
```

Reply:
Yes, will do ...